activesupport/test/cache/behaviors/cache_increment_decrement_behavior.rb

```diff
@@ -31,6 +31,18 @@ def test_decrement
     assert_equal @cache.is_a?(ActiveSupport::Cache::MemCacheStore) ? 0 : -100, missing
   end
 
+  def test_read_counter_and_write_counter
+    key = SecureRandom.uuid
+    @cache.write_counter(key, 1)
+    assert_equal 1, @cache.read(key, raw: true).to_i
+
+    assert_equal 1, @cache.read_counter(key)
+    assert_equal 2, @cache.increment(key)
+    assert_equal 2, @cache.read_counter(key)
+
+    assert_nil @cache.read_counter(SecureRandom.alphanumeric)
```

Comment:
won't this be 0 via `nil.to_i` ?

Reply:
Yeah I just pushed a fix, now it's `nil&.to_i`.